### DISCUSSION TITLE
Video feed from Mini

### DISCUSSION THREAD

**Discussion by Nitromorph** [RosettaDrone/rosettadrone#161] [Q&A]
I have successfully gotten rosetta loaded on an android phone and connected to qgroundcontrol on a laptop to the point where I can send elevation and location commands to a DJI Mini. I have yet to run full missions as I am just trying to get a feel for this. I cannot seem to get a functioning video feed to QGC. It works in rosetta, although the aspect ratio is very warped on my old Note 9. Not a big deal. I have "Enable video feed" turned on for rosetta. From what I understand this will output an H.264 UDP port 5600 stream to QGC without further configuration? On the QGC side I have video source set to UDP h.264 Video Stream on port 5600. I opened port 5600 incoming, also tested with firewall off just in case I messed up the firewall config. I saw that video feed has issues with some models. The Mini is on the tested drone list but I have not been able to figure out if I should be expected a functioning video feed, if I am doing something wrong, or if I should not currently expect a functioning video feed from this model.

**Comment by kripper** [comment]
See https://github.com/RosettaDrone/rosettadrone/issues/131
